lib/org/chromium/webidl/URLExtractor.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Let's switch the `postSet`s in this file to expressions:

```js
{ name: 'foo' },
{ name: 'barDepsOnFoo_', expression: function(foo) { /* ... */ return barDepsOnFooValue; } }
```

The name of the parameter on the expression's `function` must match the property name(s) it depends on.